### DISCUSSION TITLE
Documentation: API Documentation link fails

### DISCUSSION THREAD

**Discussion by dershver** [fo-dicom/fo-dicom#1689] [Q&A]
I built the Documentation project. When I run it, the Usage and Developer tabs on top are usable. However, the API Documentation link fails. It displays "https://localhost:44319/api/FellowOakDicom.html" link cannot be found. Any ideas how to rectify this issue ?

**Comment by mrbean-bremen** [comment]
Just stumbled over this old question...

The reason that this doesn't work is that the API documentation is not created automatically by the _Documentation_ project.
To create the API documentation, you need [docfx](https://github.com/dotnet/docfx) installed. In the documentation directory, you can then run:
```
docfx v5\docfx.json
```
This will create the documentation, and the next time you run the documentation project, it shall be shown in the browser. Obviously, you need to re-run this if you want to see changes in the API documentation that you made.
